plugin/index.js

```diff
@@ -81,6 +81,7 @@ ESLintPluginGenerator.prototype.generate = function generate() {
     this.template("_plugin.js", "lib/index.js");
     this.template("_package.json", "package.json");
     this.template("_README.md", "README.md");
+    this.template("../../.eslintrc.yml", ".eslintrc.yml");
```

Comment:
Per my comment on the issue, I think this is too opinionated. I'd be more interested in seeing if we could run `eslint --init` somehow in the process, or at least allowing the user to choose a different configuration.